docs/language/src/builtins/widgets.md

```diff
@@ -40,6 +40,7 @@ A simple button. Common types of buttons can also be created with [`StandardButt
 -   **`icon`** (_in_ _image_): The image to show in the button. Note that not all styles support drawing icons.
 -   **`pressed`**: (_out_ _bool_): Set to true when the button is pressed.
 -   **`text`** (_in_ _string_): The text written in the button.
+-   **`primary`** (_in_ _bool_): If set to `true` the button is displayed with the primary accent color (default: false).
```

Comment:
I don't mind using `true` here instead of true, but then I think it should be done consistently throughout the file. Just two lines above it's used without back-ticks :)

Reply:
In most of the file it was used without ``, I removed the `` for true in the file.